linkage/pvs_like_case_study/pvs_like_case_study_sample_data.ipynb

```diff
@@ -15,26 +15,161 @@
    "metadata": {},
```

Comment:
<p>typesetting is not working well in github for this quote block</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research_prl/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>